.github/workflows/node.js.yml

```
@@ -135,6 +135,24 @@ jobs:
           npm run test:reuse:nonUi5
 
   test-util:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: run tests
+        run: |
+          npm ci
+          npm run update-chromedriver
+          npm run test:reuse:util
+
+  test-util-data-privacy:
+    if: github.repository != github.event.pull_request.base.repo.full_name
```

I guess we have to reverse this logic sine they have to be executed for non forks. At this PR they are not
```suggestion
    if: github.repository == github.event.pull_request.base.repo.full_name
```